src/git-version.cr

```diff
@@ -139,7 +146,17 @@ module GitVersion
       major = false
       get_commits_since(previous_tag).each do |c|
         commit = c.downcase
-        if commit.includes?(MAJOR_BUMP_COMMENT)
+        match = false
+        if @major_id_is_regex
+          if /#{@major_identifier}/.match(commit)
+            match = true
+          end
+        else
+          if commit.includes?(@major_identifier)
+            match = true
+          end
+        end
```

Comment:
I am not an expert in Crystal but I was wondering if this code could not be simplified by using he if returns. At least in ruby that should be possible.

```suggestion
        match = if @major_id_is_regex
          /#{@major_identifier}/.match(commit)
        else
          commit.includes?(@major_identifier)
        end
```

Reply:
Im not so sure how that works with readability, but feel free to commit it to the branch.

Reply:
I am not sure if it works. I did it in the browser. I would have to check it. But that will take a bit more time since I have to checkout and compile it.

Reply:
So it works, but we're still left with the whole readability situation

Reply:
I think this way it is a bit more readable since you avoid the double is and just invoke one of the functions to match.
Thanks for checking. i will go forward with the pr